packages/cozy-stack-client/src/SharingCollection.js

```diff
@@ -19,8 +19,48 @@ class SharingCollection extends DocumentCollection {
       data: resp.data.map(normalizeSharing)
     }
   }
+  /**
+   *
+   * Creates a new Sharing. See https://docs.cozy.io/en/cozy-stack/sharing/#post-sharings
+   *
+   * @param {object} params
+   * @param {object} params.document The document to share
+   * @param {string} params.description Description of the sharin
+   * @param {string} params.previewPath The path (route) to see the preview
+   * @param {Array} params.rules The rules defined to the sharing. See https://docs.cozy.io/en/cozy-stack/sharing-design/#description-of-a-sharing
+   * @param {Array} params.recipients Recipients to add to the sharings (will have the same permissions given by the rules defined by the sharing )
+   * @param {Array} params.readOnlyRecipients Recipients to add to the sharings with only read only access
```

Comment:
@typedef Recipient ?

Reply:
It seems we can't import `typedef` from other file. Since `Recipient` is an `io.cozy.contacts` document I wanted to create a typedef Contact in ContactCollection or ContactModel and import it there but I can't? 

So do you have an idea of how we can re-use typedef definition ? Or do you have an advice on how to do?

Reply:
It seems that importing a type is only possible in a typescript context. I have also seen https://www.npmjs.com/package/jsdoc-import-typedef that could be good to check out. In the context of this PR, I have no proper solution right now, maybe just writing `@param {Array<Recipient>}` right now is sufficient. 

For the future, I think we definitely have to check if we could use typescript tools to generate our doc instead of jsdoc.